.dockerignore

```diff
@@ -12,6 +12,5 @@ Dockerfile
 .editorconfig
 .gitattributes
 .gitignore
-.travis.yml
```

Comment:
don't we use travis? it runs checks

Reply:
We used CI system run checks. Travis old think and don't supported yet.